WordPress/Sniffs/VIP/RestrictedFunctionsSniff.php

```diff
@@ -283,6 +283,15 @@ public function getGroups() {
 				),
 			),
 
+			// @link https://vip.wordpress.com/documentation/change-your-pretty-permalinks-or-add-custom-rewrite-rules/
+			'flush_rewrite_rules' => array(
+				'type'     => 'warning',
+				'message'   => '%s Using flush_rewrite_rules() is not required on WordPress.com VIP since it is done on every deploy and every theme switch. Using this will cause excessive writes to the database and slow down performance',
```

Comment:
The error outputted will be
> flush_rewrite_rules Using flush_rewrite_rules() is not required on WordPress.com VIP since it is done on every deploy and every theme switch. Using this will cause excessive writes to the database and slow down performance

What about
> Using %s() is not needed on WordPress.com VIP since it is done on every deploy and every theme switch. Using this will cause excessive writes to the database and slow down performance.

Make the function dynamic, instead of required use needed or necessary, add a fullstop to the end of the second sentence.